From ffa92ae6d4148b38d229fc2aa12c1ee4980f20d5 Mon Sep 17 00:00:00 2001
From: "A.Dorn" <dorna8550@gmail.com>
Date: Tue, 22 Aug 2023 20:55:37 +0200
Subject: [PATCH] sun4i/Lima : Mesa/Lima - Display issues on Sunxi-Boards

Signed-off-by: A.Dorn <dorna8550@gmail.com>
---
 drivers/soc/sunxi/sunxi_mbus.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/soc/sunxi/sunxi_mbus.c b/drivers/soc/sunxi/sunxi_mbus.c
index 1734da357..8c9bbe8a6 100644
--- a/drivers/soc/sunxi/sunxi_mbus.c
+++ b/drivers/soc/sunxi/sunxi_mbus.c
@@ -33,6 +33,7 @@ static const char * const sunxi_mbus_devices[] = {
 	"allwinner,sun4i-a10-display-backend",
 	"allwinner,sun4i-a10-display-frontend",
 	"allwinner,sun4i-a10-video-engine",
+	"allwinner,sun4i-a10-mali",
 	"allwinner,sun5i-a13-display-backend",
 	"allwinner,sun5i-a13-video-engine",
 	"allwinner,sun6i-a31-csi",
@@ -41,8 +42,10 @@ static const char * const sunxi_mbus_devices[] = {
 	"allwinner,sun7i-a20-display-backend",
 	"allwinner,sun7i-a20-display-frontend",
 	"allwinner,sun7i-a20-video-engine",
+	"allwinner,sun7i-a20-mali",
 	"allwinner,sun8i-a23-display-backend",
 	"allwinner,sun8i-a23-display-frontend",
+	"allwinner,sun8i-a23-mali",
 	"allwinner,sun8i-a33-display-backend",
 	"allwinner,sun8i-a33-display-frontend",
 	"allwinner,sun8i-a33-video-engine",
-- 
2.39.2

